media/robots.txt

```diff
@@ -6,5 +6,30 @@ Request-rate: 1/5
 Disallow: /*feed=rss
 Disallow: /*type=feed
 Disallow: /skins
-Disallow: /template:
+Disallow: /*docs/feeds
+Disallow: /*docs/templates
+Disallow: /*docs*Template
 Disallow: /media
+Disallow: /*docs*$compare
+Disallow: /*docs*$revision
+Disallow: /*docs*$history
+Disallow: /*docs*$edit
+Disallow: /*docs*$children
+Disallow: /*docs*$flag
+Disallow: /*docs*$translate
+Disallow: /*docs*$locales
+Disallow: /*docs*$json
+Disallow: /*docs*$styles
+Disallow: /*docs*$toc
+Disallow: /*docs*$move
+Disallow: /*docs*$quick-review
+Disallow: /*docs*$revert
+Disallow: /*docs*$repair_breadcrumbs
+Disallow: /*docs*$delete
+Disallow: /*docs*$restore
+Disallow: /*docs*$purge
+Disallow: /*docs*$subscribe
+Disallow: /*docs*$vote
+Disallow: /*docs*.json
```

Comment:
will this block https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/package_json ?